tests/test_pulser_pasqal.py

```diff
@@ -410,6 +410,14 @@ def test_submit(fixt_pasqal_cloud, parametrized, emulator, mimic_qpu, seq, mock_
     remote_results = fixt_pasqal_cloud.pasqal_cloud.submit(
         seq, job_params=job_params, batch_id="open_batch"
     )
+    assert remote_results.get_available_results() == {
```

Comment:
I tried to reproduce the bug in the test by adding this, but somehow the test passes. Do you know why?